waspc/src/Wasp/Generator/SdkGenerator/JobGenerator.hs

```diff
@@ -36,4 +37,12 @@ genJobType (jobName, job) =
           "entities" .= maybe [] (map (makeJsonWithEntityData . AS.refName)) (J.entities job)
         ]
 
-    jobExecutorTypesImportPath = jobExecutorTypesImportPathFromSdk (J.executor job)
+    jobExecutorTypesImportPath = getJobExecutorTypesImportPath (J.executor job)
+
+getImportPathForJobName :: String -> Path Posix (Rel d) File'
+getImportPathForJobName jobName = makeSdkImportPath $ [reldirP|jobs|] </> fromJust (SP.parseRelFileP jobName)
+
+-- | We are importing relevant types per executor e.g. JobFn, this functions maps
+--   the executor to the import path of the relevant types.
+getJobExecutorTypesImportPath :: JobExecutor -> Path Posix (Rel r) File'
+getJobExecutorTypesImportPath PgBoss = makeSdkImportPath [relfileP|jobs/pgBoss/types|]
```

Comment:
I moved this here because that's how the SDK generator is currently organized (we import the necessary stuff into framework generators).

Reply:
Nitpick: We are missing the new line at the end of the file